src/index.html

```diff
@@ -54,6 +54,12 @@
 
     <h1>Lotus</h1>
 
+    <h2>Coming soon!</h2>
+
+    <p>The Lotus app is a tool for linguists to manage their linguistic data, including lexicons, texts, and cognate sets. Keep checking back for updates!</p>
+    <p><a href="https://digitallinguistics.io" rel="nofollow">Learn more about the Digital Linguistics project here!</a></p>
+    <p><a href="https://twitter.com/digitalling" rel="nofollow">Follow Digital Linguistics on Twitter!</a></p>
```

Comment:
Good you're following best practice with `nofollow` (and `noopener` is another good one to use). This happens to be the exact case where `nofollow` isn't necessary though: when you endorse the content of the places you're linking to. Since both of these links go to official DLx content, we should remove the `nofollow` value. We want DLx to get SEO credit for these links.

```suggestion
    <p><a href=https://digitallinguistics.io>Learn more about the Digital Linguistics project here!</a></p>
    <p><a href=https://twitter.com/digitalling>Follow Digital Linguistics on Twitter!</a></p>
```